.github/workflows/ci.yml

```diff
@@ -53,7 +53,7 @@ jobs:
       - uses: actions/checkout@v3
       - uses: ./.github/actions/setup
         with:
-          use_lockfile: false
+          use_lockfile: true
```

Comment:
this is temporary to show CI would be green if not for the meddling ts getting in the way

Reply:
What do you mean by temporary? is it something that we need to remove in a few weeks when something upstream is fixed? do we want to merge it with this and create an issue to remove it? I assume we'll discuss more at the meeting 😂

Reply:
meaning without this any PR will be very bloody right now
turned this on as a means to validate that the pr itself with the changes passes, without interference from floating packages (@babel/types) that have nothing to do with this bugfix or change